federation/apis/openapi-spec/logs.json

```diff
@@ -0,0 +1,46 @@
+{
+  "swagger": "2.0",
+  "info": {
+   "title": "Generic API Server /logs",
+   "version": "unversioned"
+  },
+  "paths": {
+   "/logs/": {
+    "get": {
+     "schemes": [
+      "https"
+     ],
+     "operationId": "logFileListHandler",
+     "responses": {
+      "default": {
+       "description": "Default Response."
```

Comment:
If a route does not provide a response, a default response will be used (provided as config to the spec generator too). We can change this for federation or kubernetes, what do you think it should be?